homeassistant/components/zha/device_action.py

```diff
@@ -78,13 +78,10 @@
     service_name = SERVICE_NAMES[action_type]
     zha_device = await async_get_zha_device(hass, config[CONF_DEVICE_ID])
 
-    service_action = {
-        service.CONF_SERVICE: "{}.{}".format(DOMAIN, service_name),
-        ATTR_DATA: {ATTR_IEEE: str(zha_device.ieee)},
-    }
+    service_data = {ATTR_IEEE: str(zha_device.ieee)}
 
-    await service.async_call_from_config(
-        hass, service_action, blocking=True, variables=variables, context=context
+    await hass.services.async_call(
+        DOMAIN, service_name, service_data, blocking=True, context=context
```

Comment:
@dmulcahey Is this change OK?

Reply:
I’d assume yes if the tests pass unless I did something silly. I think it should be fine.